.github/workflows/npm-publish-artillery-canary.yml

```diff
@@ -5,34 +5,45 @@ on:
       - main
     paths:
       - 'packages/artillery/**'
-      - 'packages/artillery-plugin-expect/**'
-      - 'packages/artillery-plugin-publish-metrics/**'      
-      - 'packages/artillery-plugin-metrics-by-endpoint/**'
+      - 'packages/artillery-engine-playwright/**'
```

Comment:
reordered this to be in the same order as packages in the repo. Also added missing packages.